plugin.video.francetv/addon.py

```diff
@@ -0,0 +1,272 @@
+# Copyright (C) 2018 melmorabity
```

Comment:
It is a general good practice to keep the entrypoint python file to a minimum since it won't get compiled to bytecode by the python interpreter. Usually you store the main file in the `resources` folder and just call it from the entrypoint:

```python
from resources.lib import addon

addon.run()
```

Do you want to take the oportunity to implement something like this and improve your addon performance or do you want to keep it for a future update?

Regards

Reply:
@enen92 Thaks for your comment, I will update the add-on with your suggestions.